node/src/common/JXCoreEngine.cpp

```diff
@@ -0,0 +1,456 @@
+
+#include <cstdlib>
+#include <functional>
+#include <memory>
+#include <mutex>
+#include <queue>
+#include <string>
+#include <thread>
+#include <unordered_map>
+
+#include "Log.h"
+#include "INodeEngine.h"
+#include "AsyncQueue.h"
+#include "WorkItemDispatcher.h"
+#include "JXCoreEngine.h"
+
+#include "jxcore/jx.h"
```

Comment:
The "external" directory is for stuff that has been copied from external sources. But I wrote JXCoreEngine.cpp specifically for this repo.

---
In reply to: [71265341](https://github.com/openT2T/client/pull/1#discussion_r71265341) [](ancestors = 71265341)